examples/android/RunAnywhereAI/app/src/main/java/com/runanywhere/runanywhereai/presentation/models/ModelSelectionBottomSheet.kt

```
@@ -64,757 +94,556 @@ fun ModelSelectionBottomSheet(
+        name.contains("liquid") -> R.drawable.liquid_ai_logo
+        name.contains("piper") -> R.drawable.hugging_face_logo
+        name.contains("whisper") -> R.drawable.hugging_face_logo
+        name.contains("sherpa") -> R.drawable.hugging_face_logo
+        else -> R.drawable.hugging_face_logo
     }
 }
 
-// DeviceInfoRow: Label + Spacer + Text(value).foregroundColor(AppColors.textSecondary)
+private fun formatBytes(bytes: Long): String {
+    val gb = bytes / (1024.0 * 1024.0 * 1024.0)
+    return if (gb >= 1.0) String.format("%.2f GB", gb) else String.format("%.0f MB", bytes / (1024.0 * 1024.0))
```

detekt (1.23.8)

[warning] String.format("%.2f GB", gb) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)